tests/test_lib_utils.py

```diff
@@ -822,6 +822,8 @@ def test_28_yubikey_utils(self):
         self.assertEqual(modhex_decode('cbdefghijklnrtuv'),
                           binascii.unhexlify('0123456789abcdef'))
         self.assertEqual(modhex_decode('fjhbhrhrhv'), b'Hallo')
+        # fail with invalid modhex
+        self.assertRaises(binascii.Error, modhex_decode, 'nlltvcc')
```

Comment:
We should do the same here:
`self.assertRaises((binascii.Error, TypeError), modhex_decode, 'nlltvcc')`

Reply:
Done in 0bba2428